src/environments/environment.prod.ts

```diff
@@ -15,4 +15,5 @@ export const environment = {
   email: undefined,
   DB_PROXY_PREFIX: "/db",
   DB_NAME: "app",
+  firebaseConfig: {} as any,
```

Comment:
can you create an interface to explicitly type this object?

Also, let's name it more framework-agnostic "notificationsConfig"?